### ISSUE TITLE
Simplify some keys of pipeline.yml

### ISSUE THREAD

**Issue by mizzy** [walter-cd/walter#81]
It would be better to change `stage_name` and `stage_type` to `name` and `type` for readability and writeability like this.

```yaml
 pipeline:
      - name: command_stage_1
         type: command
         command: echo "hello, world"
      - name: command_stage_2
         type: command
         command: echo "hello, world, command_stage_2"
      - name: command_stage_3
         type: command
         command: echo "hello, world, command_stage_3"
```

How do you think?

**Comment by takahi-i** [comment]
Agree!

**Comment by takahi-i** [comment]
Fixed with #88!
